packages/funding-rate-proposer/package.json

```diff
@@ -8,7 +8,7 @@
     "@uma/financial-templates-lib": "^2.0.1",
     "async-retry": "^1.3.1",
     "dotenv": "^6.2.0",
-    "truffle": "^5.1.35"
+    "truffle": "^5.2.3"
```

Comment:
Does truffle need to be a regular dep in this package? I don't know if it _ever_ needs to be a regular dependency... unless there's a `require("truffle")` floating around...?

Reply:
agreed yea was wondering if it should be a dev dep

Reply:
This is a great point, i'll try moving all truffle imports to devDeps